ere-guests/stateless-validator/sp1/Cargo.toml

```diff
@@ -1,32 +1,35 @@
 [package]
 name = "succinct-guest"
-version.workspace = true
-edition.workspace = true
-rust-version.workspace = true
-license.workspace = true
+version = "0.1.0"
+edition = "2024"
+rust-version = "1.85"
+license = "MIT OR Apache-2.0"
+
 
 [dependencies]
-sp1-zkvm.workspace = true
-reth-stateless.workspace = true
-reth-ethereum-primitives = { workspace = true, features = [
+sp1-zkvm = "5.0.5"
+reth-stateless = { git = "https://github.com/kevaundray/reth", rev = "03364a836774c72f4e354de924330fee6a41be68" }
+reth-ethereum-primitives = { git = "https://github.com/kevaundray/reth", rev = "03364a836774c72f4e354de924330fee6a41be68", features = [
     "serde",
     "serde-bincode-compat",
 ] }
-reth-primitives-traits = { workspace = true, features = [
+reth-primitives-traits = { git = "https://github.com/kevaundray/reth", rev = "03364a836774c72f4e354de924330fee6a41be68", features = [
     "serde",
     "serde-bincode-compat",
 ] }
-reth-evm-ethereum = { workspace = true }
-reth-chainspec = { workspace = true }
+reth-evm-ethereum = { git = "https://github.com/kevaundray/reth", rev = "03364a836774c72f4e354de924330fee6a41be68" }
+reth-chainspec = { git = "https://github.com/kevaundray/reth", rev = "03364a836774c72f4e354de924330fee6a41be68" }
```

Comment:
This was the "ugly" part of this story, since should be in sync with:
- `ere_hosts` since it is the one generating the proofs. So ideally guest programs SDK match the prover version.
- `ere` since it is the one installing the compilation chain for the guest program

@kevaundray, [mentioned](https://github.com/eth-act/ere/pull/54#issuecomment-3074353585) some potential idea to potentially generalize the way guest programs are created to reduce the burden. e.g., all `stateless-validator` guest program use the same `reth-*` dependencies and main guest program implementation.